tools/experiments/experiments.html

```diff
@@ -212,6 +212,11 @@ <h2>Release channels</h2>
   </section>
 
   <section class="content">
+      <p>
+        <em>Note:</em> these controls set a cookie on cdn.ampproject.org. Depending on your browser settings, you might need
+        to disable cross-site tracking / enhanched tracking protection or white-list this domain name for the opt-in cookie
```

Comment:
To make this more actionable, I'd capitalize "Enhanced Tracking Protection" and link to the specific browser documentation.

Also "enhanched".

Reply:
+1 to the suggestion above.

Reply:
Instead of `opt-in cookie`, would it be clearer to say `the AMP_CANARY cookie`?

Reply:
`AMP_CANARY` is the implementation details, "opt-in cookie" is the thing we're discussing :)